controller/backup_target_controller.go

```diff
@@ -546,22 +553,48 @@ func (btc *BackupTargetController) getInfoFromBackupStore(backupTarget *longhorn
 	return info, nil
 }
 
-func (btc *BackupTargetController) syncBackupVolume(backupTarget *longhorn.BackupTarget, backupStoreBackupVolumeNames []string, syncTime metav1.Time, log logrus.FieldLogger) error {
-	backupStoreBackupVolumes := sets.New[string](backupStoreBackupVolumeNames...)
-
+func (btc *BackupTargetController) cleanupDuplicateBackupVolumes(backupTargetName string, log logrus.FieldLogger) (map[string]*longhorn.BackupVolume, error) {
 	// Get a list of all the backup volumes that exist as custom resources in the cluster
-	clusterBackupVolumes, err := btc.ds.ListBackupVolumesWithBackupTargetNameRO(backupTarget.Name)
+	clusterBackupVolumes, err := btc.ds.ListBackupVolumesWithBackupTargetNameRO(backupTargetName)
```

Comment:
What do we need the prefix `cluster`?
```suggestion
	clusterBackupVolumes, err := btc.ds.ListBackupVolumesWithBackupTargetNameRO(backupTargetName)
```

Reply:
Yes, we don't need it. Modified. Thanks.